controller/src/main/java/org/jboss/as/controller/operations/global/ReadAttributeHandler.java

```diff
@@ -149,24 +149,27 @@ void doExecute(OperationContext context, ModelNode operation, FilteredData filte
 
     private void doExecuteInternal(OperationContext context, ModelNode operation) throws OperationFailedException {
         validator.validate(operation);
+        final ImmutableManagementResourceRegistration registry = context.getResourceRegistration();
         String attributeName = GlobalOperationAttributes.NAME.resolveModelAttribute(context, operation).asString();
         final boolean defaults = GlobalOperationAttributes.INCLUDE_DEFAULTS.resolveModelAttribute(context,operation).asBoolean();
-        final boolean useEnhancedSyntax = containsEnhancedSyntax(attributeName);
-        String attributeExpression = attributeName;
+        boolean useEnhancedSyntax = containsEnhancedSyntax(attributeName);
```

Comment:
Also with that kind of change you would also address AbstractCollectionHandler that uses EnhancedSyntaxSupport as well. and would need similar change otherwise.